### PR TITLE
Update homepage layout with new IA

### PR DESCRIPTION
**Pull request by juliaelman**
⚠️ **The PR is downstream from https://github.com/department-of-veterans-affairs/developer-portal/pull/91 and should NOT be merged in until that has been merged.**

This pull request is for small updates to the layout of the homepage of the developer portal. They are based on the IA changes that happened in https://github.com/department-of-veterans-affairs/developer-portal/pull/85.

See screencast below for reference:

bLpDzNN5xE

### CODE REVIEW

**Comment by juliaelman** [comment]
@jeff2d2 updated here caebc31

Screenshot FYR

Screen Shot 2019-03-22 at 1 57 28 PM

File: src/containers/Home.tsx [hunk 1]
```diff
@@ -1,82 +1,89 @@
...
-                <div className="usa-grid">
-                  <div className="usa-width-one-half">
-                    <div className="video-wrapper">
-                      <iframe src="https://www.youtube.com/embed/at69_Uz05vA" frameBorder="0" allowFullScreen={true} />
-                    </div>
                   </div>
                 </div>
               </section>
+              <div className="usa-grid">
+                <div className="ruled-stars">{null}</div>
```
Comment (kalilsn) [inline src/containers/Home.tsx]: what's the `{null}`? was one of our linters complaining about the div not having content?

File: src/containers/Home.scss [hunk 2]
```diff
@@ -63,11 +51,48 @@
...
-        top: 20px;
-        left: 0;
-        width: 100%;
-        height: 100%;
+          display: block;
+          position: absolute;
+          top: 20px;
+          left: 0;
+          width: 100%;
+          height: 100%;
```
Comment (kalilsn) [inline src/containers/Home.scss]: looks like this block got overindented a bit

File: src/containers/Home.tsx [hunk 3]
```diff
@@ -1,82 +1,89 @@
...
-              <section role="region" aria-label="API Top-Level List" className="usa-section api-section">
+
+              <section role="region" aria-label="Page Hero" className="usa-hero">
                 <div className="usa-grid">
-                  <h2>Explore VA APIs</h2>
-                  <p className="usa-font-lead">
-                    We want to empower our partners to build innovative solutions for Veterans. So we designed a framework to deliver a modern API development experience and rolled out our first set of standards-based APIs.
-                  </p>
+                  <div className="usa-hero-callout">
+                    <h1 className="">Put VA Data to Work</h1>
```
Comment (kalilsn) [inline src/containers/Home.tsx]: I'm guessing this `className` can just be deleted!

File: src/base.scss [hunk 4]
```diff
@@ -15,6 +15,11 @@ $medium-large-screen: 768px;
 $medium-screen: 640px;
 $large-screen: 951px;
 
+.usa-grid {
+  max-width: 1200px !important;
+  margin-left: 0 !important;
```
Comment (kalilsn) [inline src/base.scss]: can you add a comment about why these are `!important`?

File: src/containers/Home.tsx (hunk 1 above)
Comment (juliaelman) [reply src/containers/Home.tsx]: Yep.

File: src/containers/Home.scss (hunk 2 above)
Comment (juliaelman) [reply src/containers/Home.scss]: Fixed here 900320d

File: src/containers/Home.tsx (hunk 3 above)
Comment (juliaelman) [reply src/containers/Home.tsx]: Fixed here ae58a03

File: src/base.scss (hunk 4 above)
Comment (juliaelman) [reply src/base.scss]: Because it is an important override of the USWDS.

**Pull request merged**